### ISSUE TITLE
Reword logout message

### ISSUE THREAD

**Issue by nbradbury** [wordpress-mobile/WordPress-Android#7655]
As discussed elsewhere, our logout confirmation is too wordy and is scary to some users. 

If there are no local changes, the logout confirmation should be simply, "Log out of WordPress?"

If there **are** unsaved changes, the message should be, "You have changes to X posts that haven’t been uploaded to your site. Logging out now will delete those changes. Log out anyway?"

Note that the wording has been approved by Editorial already.

**Comment by nellofn** [comment]
> Logging out now will delete those changes.

I dig the shorter message. However, can we at least add "from your device"?

I won't expect the user to be fully aware that changes are not going to be deleted from the server as well.
